vent/core/rq_worker/watch.py

```diff
@@ -55,6 +57,18 @@ def file_queue(path, template_path="/vent/"):
         else:
             files = '/'
 
+        # deal with ~
+        files = os.path.expanduser(files)
+
+        file_name = ''
+        # escape any funky symbols to allow users FREEDOM of directory name
+        for char in files:
+            if char in set(punctuation):
```

Comment:
Oh I think that's a special case. I think to get a literal `\`, you have to have `\\`